### ISSUE TITLE
Feature request: Add duration information to Output 1 and 2

### ISSUE THREAD

**Issue by CirruZZ** [ottopaulsen/node-red-contrib-power-saver#202]
It would be neat to have the duration info of the upcoming window when Output 1 and Output 2 fires. This applies s to to all nodes that have Output 1 and Output 2 for sending ON/OFF TRUE/FALSE.

As now it sends 
`{"payload":"ON"}`

Suggestion would be to add 
`{"payload":"ON", "countHours": 3}`

This information is available in `payload.schedule` of Output 3 but it would be nice to have directly when receiving output from Output 1 and Output 2.

**Comment by ottopaulsen** [comment]
Good idea. I think the msg object should look something like this (but JSON):
```
{ 
  payload: "ON",
  data: {
           countHours: 3
  }
}

**Comment by CirruZZ** [comment]
Sorry, I just saw your reply now. Absolutely, that loojs great, as long as you get the information, I'm happy.
